README.md

```diff
@@ -15,6 +15,24 @@ The library already implements enough features to enable developers to build fun
 Documentation is available [here](https://ariadne.readthedocs.io/).
 
 
+## Features
+
+- Simple, quick to learn and easy to memorize API.
+- Compatibility with GraphQL.js version 14.0.2.
+- Queries, mutations and input types.
+- Asynchronous resolvers and query execution.
+- Custom scalars and enums.
+- Defining schema using SDL strings.
+- Loading schema from `.graphql` files.
+- GraphQL syntax validation and colorization via `gql()` helper function.
```

Comment:
Should this link to the Apollo plugin? Not sure if it's a feature of Ariadne

Reply:
We are linking to Apollo plugin few lines below that one, so maybe we should reorder/reword those?